ibis/pyspark/compiler.py

```diff
@@ -343,110 +346,77 @@ def fn(col):
 
 @compiles(ops.Any)
 def compile_any(t, expr, scope, context=None, **kwargs):
-    def fn(col):
-        if 'window' in kwargs:
-            window = kwargs['window']
-            return F.max(col).over(window)
-        else:
-            return F.max(col)
-
-    return compile_aggregator(t, expr, scope, fn, context, **kwargs)
+    return compile_aggregator(t, expr, scope, F.max, context, **kwargs)
```

Comment:
@icexelloss could you provide a docstring here using numpy format? https://numpydoc.readthedocs.io/en/latest/format.html

Reply:
Thanks for the review. I like docs but I feel like docstring for these methods won't be very much value because it will pretty much be copy/paste of each other...sth like

```
Compile XXX operator.

t: Translator
expr: The expression to translate
scope: Scope for the translation
context: Aggregation context
```
I think it might be better to put some doc at the top of this file that explains these compile methods in general. WDYT?

Reply:
As a `public` function it should be very well documented.

but as it is not used directly by an end user you can change these functions to be private, eg:

`def compile_any(..):` -> `def _compile_any(...):`

and the documentation for these functions doesn't need to be so complete.
 
and you can check the docstrings using `pydocstyle ibis/pyspark/compiler.py`

I don't mind to have "copy/paste" documentation as well for the private functions .. everything that could help the development is very welcome. but for now if you change that functions from public to private .. that works fine.

Reply:
@xmnlab we don't normally document these types of operations because the signature is the same; these are not public functions for a user, rather implementations so only developers see these. happy to have a generalized documentation template, but adding copy-pasting is not great here.